src/wp-includes/media.php

```diff
@@ -1974,6 +1970,35 @@ function wp_img_tag_add_loading_optimization_attrs( $image, $context ) {
 		$context
```

Comment:
You need to pass the `$decoding_val` to the `wp_get_loading_optimization_attributes()` call, so that the function can respect any potentially existing `decoding` attribute.

Reply:
Thanks. This was factored in in the last update.